### ISSUE TITLE
Find... feature doesn't see plugins

### ISSUE THREAD

**Issue by TheresaSchmidt** [dialogos-project/dialogos#174]
**Describe the bug**
I observed that when I search for a variable, plugin nodes using this variable are not found. Also, they are not listed in the drop-down for the search for node types. They are found, however, when searching for the nodes' names.

**To Reproduce**
Please attach a minimal example dialog exposing the bug if applicable.
1. Example: [find.zip](https://github.com/dialogos-project/dialogos/files/2825737/find.zip)
2. Go to menu ```Nodes > Find...``` or use ```Strg + F``` to open the search mask.
3. Select ```Uses variable```and type in ```test```.
4. ```test``` is found only in the variable setting node, although it is also used in the speech recognition node as well as in the speech synthesis node.

**Expected behavior**
The search should find every appearance of the variable in the whole dialogue. Otherwise it's not helpful.

**Installation information**
 - OS: Windows
 - Version 2.0.5]
 - Installer

**Comment by timobaumann** [comment]
there's a `find` functionality in DialogOS?

**Comment by akoehn** [comment]
I also did not know that.  I think that this functionality has to be somehow implemented by the nodes and it is unclear to me whether it is actually desirable to search for content.  Should Script/Groovy nodes search their code? Should recognizer nodes search their recognized words or grammar?

**Comment by TheresaSchmidt** [comment]
Yes, it very desirable to complete and enhance the find functionality! We would also really appreciate it if you could enter just any kind of text and it searched everywhere (node names, node descriptions, grammars, patterns, notes, ...).
